config/locales/en.yml

```diff
@@ -21,6 +21,9 @@
 en:
   # Overridden here due to a bug in spree i18n (Issue #870, and issue #1800)
   language_name: "English" # Localised name of this language
+  time:
+    formats:
+      long: "%B %d, %Y%l:%M %p" # %l adds a single space to the left
```

Comment:
> %l adds a single space to the left

Actually, this is a padding that is only included on single-digit numbers ([refer](https://apidock.com/ruby/DateTime/strftime)). For a two-digit number there would be no space, eg:

```
DateTime.new(2023, 10, 13, 12, 00).strftime "%B %d, %Y%l:%M %p"
=> "October 13, 202312:00 PM"
```

So if we want to strip the blank space, we can use the `-` flag mentioned in the docs, like this: `%-l`

Reply:
> > %l adds a single space to the left
> 
> Actually, this is a padding that is only included on single-digit numbers ([refer](https://apidock.com/ruby/DateTime/strftime)). For a two-digit number there would be no space, eg:
> 
> ```
> DateTime.new(2023, 10, 13, 12, 00).strftime "%B %d, %Y%l:%M %p"
> => "October 13, 202312:00 PM"
> ```
> 
> So if we want to strip the blank space, we can use the `-` flag mentioned in the docs, like this: `%-l`

That's nice :heart: . Thanks for letting me know :smile: